apps/mobile/src/libs/enums/fonts/font-size.enum.ts

```diff
@@ -0,0 +1,27 @@
+const FontSize = {
```

Comment:
Should be in separate filed, in ~/libs/styles

Reply:
I didn't get it. Do I need to move FontSize to the styles folder?

Reply:
Yes, but not only FontSize, but other ones as well